compiler/z/codegen/OMRMemoryReference.cpp

```diff
@@ -2076,9 +2077,7 @@ OMR::Z::MemoryReference::enforce4KDisplacementLimit(TR::Node * node, TR::CodeGen
    if (self()->ignoreNegativeOffset())
       return preced;
 
-   TR_ASSERT(!forcePLXFixup, "This logic is only used for markAndAdjustForLongDisplacementIfNeeded. You probably want the other flag.");
```

Comment:
Remove code associated with `forcePLXFixup` since it's dead.

Reply:
Are you certain? I recall @dchopra001 tried to remove it at one point and it caused issues. I forget what the particular issue was though. Perhaps touch base with him.

Reply:
Will revert this change.

Reply:
For reference I do want it removed if we can do so without introducing bugs as it is clearly PLX specific.

Reply:
There's no issue in my testing. However, this flag is actually used in a few places so it's not safe to blindly remove it.